app/code/Magento/Cron/Model/Schedule.php

```diff
@@ -94,14 +102,16 @@ public function setCronExpr($expr)
      */
     public function trySchedule()
     {
+        $this->dateTime = $this->dateTime ?: \Magento\Framework\App\ObjectManager::getInstance()
+            ->get(\Magento\Framework\Stdlib\DateTime\DateTime::class);
         $time = $this->getScheduledAt();
         $e = $this->getCronExprArr();
 
         if (!$e || !$time) {
             return false;
         }
         if (!is_numeric($time)) {
-            $time = strtotime($time);
+            $time = strtotime($time) + $this->dateTime->getGmtOffset();
```

Comment:
We should not calculate the time for another timezone using inline code. In the future, it will be hard to track all such places if there some bugs. Please, use` \Magento\Framework\Stdlib\DateTime\TimezoneInterface` to convert to date `$time` to Default timezone.